modules/plugin/jdbc/jdbc-sqlserver/pom.xml

```diff
@@ -90,15 +90,25 @@
 			<dependencies>
 				<dependency>
 					<groupId>com.microsoft.sqlserver</groupId>
-					<artifactId>sqljdbc41</artifactId>
+					<artifactId>mssql-jdbc</artifactId>
 					<version>${sqlServerDriverVersion}</version>
 				</dependency>
+				<dependency>
```

Comment:
If these are only needed for particular cases, but not always, and are not found in repos, but have to be manually downloaded and installed, I'd mark them as optional and provide info on how to get them installed on a dev machine,
as indicated here: https://github.com/geotools/geotools/blob/master/modules/plugin/jdbc/jdbc-sqlserver/README.txt

Reply:
Never mind, it seems that Microsoft has finally seen the light and started adding stuff in the repos without click-through licenses.